src/sonic-host-services/scripts/caclmgrd

```diff
@@ -653,8 +752,21 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
             # Get the redisselect object from selectable object
             redisSelectObj = swsscommon.CastSelectableToRedisSelectObj(selectableObj)
 
-            # Get the corresponding namespace from redisselect db connector object
+            # Get the corresponding namespace and db_id from redisselect
             namespace = redisSelectObj.getDbConnector().getNamespace()
+            db_id = redisSelectObj.getDbConnector().getDbId()
+
+            if db_id == state_db_id:
+                if self.DualToR:
+                    while True:
+                        key, op, fvs = subscribe_mux_cable.pop()
+                        if not key:
+                            break
+                        self.log_info("mux cable update : '%s'" % str((key, op, fvs)))
+                        self.update_dhcp_acl(key, op, dict(fvs))
+                continue
```

Comment:
Can you confirm this flow is tested on a non-dualtor testbed?

Reply:
yes, i tested it with both subtype=dualtor or none on a vs testbed